test/Plutip/Staking.purs

```diff
@@ -113,8 +113,7 @@ suite = do
   -- We must never select this pool, because it retires at the third epoch
   -- (this is Plutip internal knowledge)
   -- https://github.com/mlabs-haskell/plutip/blob/7f2d59abd911dd11310404863cdedb2886902ebf/src/Test/Plutip/Internal/Cluster.hs#L692
-  retiringPoolId <- liftEffect
-    $ maybe (throw "unable to decode poolId bech32") pure
+  retiringPoolId <- liftEffect $ liftM (error "unable to decode poolId bech32")
```

Comment:
I don't think this `liftEffect` is necessary, due to `liftM` being over any `m` with `MonadError Error m`